bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/AsterismParamSetCodecSpec.scala

```diff
@@ -20,76 +20,7 @@ import scalaz._
 import Scalaz._
 
 
-class AsterismParamSetCodecSpec extends Specification with ScalaCheck with Arbitraries {
```

Comment:
We need all of these outside of just testing `ParamSetCodecSpec`, so I moved them to `Almosts` since there is no reason not to make them more available.